systemtest/src/main/java/io/strimzi/systemtest/resources/operator/BundleResource.java

```diff
@@ -196,6 +200,11 @@ public DeploymentBuilder buildBundleDeployment() {
         // Apply updated env variables
         clusterOperator.getSpec().getTemplate().getSpec().getContainers().get(0).setEnv(envVars);
 
+        if (SYSTEM_TEST_STRIMZI_IMAGE_PULL_SECRET != null && !SYSTEM_TEST_STRIMZI_IMAGE_PULL_SECRET.isEmpty()) {
```

Comment:
```suggestion
        if (Environment.SYSTEM_TEST_STRIMZI_IMAGE_PULL_SECRET != null && !Environment.SYSTEM_TEST_STRIMZI_IMAGE_PULL_SECRET.isEmpty()) {
```

Could you please use it as it's shown in the suggestion? We use variables from Environment this way across the whole project.

Reply:
Yes. I'll change it.

Reply:
rebased to resolve the conflicts and changed the variable references as requested